layers/core_checks/cc_cmd_buffer.cpp

```diff
@@ -322,6 +322,15 @@ bool CoreChecks::PreCallValidateBeginCommandBuffer(VkCommandBuffer commandBuffer
                                          begin_info_loc.pNext(Struct::VkDeviceGroupCommandBufferBeginInfo, Field::deviceMask),
                                          "VUID-VkDeviceGroupCommandBufferBeginInfo-deviceMask-00107");
     }
+    if ((pBeginInfo->flags & VK_COMMAND_BUFFER_USAGE_RENDER_PASS_CONTINUE_BIT) != 0) {
+        if ((cb_state->command_pool->queue_flags & VK_QUEUE_GRAPHICS_BIT) == 0) {
+            const LogObjectList objlist(commandBuffer, cb_state->command_pool->Handle());
+            skip |= LogError("VUID-VkCommandBufferBeginInfo-flags-09123", objlist, error_obj.location,
+                             " contain VK_COMMAND_BUFFER_USAGE_RENDER_PASS_CONTINUE_BIT, but the command pool the command buffer "
+                             "%s was allocated from does not support graphics operations (%s).",
```

Comment:
```suggestion
                             "%s was allocated from only supports %s.",
```

the VU in the spec explains already it requires graphics